### PR TITLE
LPS-76464 - Source formatting

### PR DESCRIPTION
**Pull request by jbalsas**
@jbalsas Please review auto_field.js

@sergiogonzalez Please review collaboration part

### CODE REVIEW

**Comment by robertoDiaz** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/collaboration/social/social-group-statistics-web/src/main/resources/META-INF/resources/add_activity_counter.jsp [hunk 1]
```diff
@@ -64,7 +64,7 @@ Collections.sort(activityCounterNames, new SocialActivityCounterNameComparator(l
 		<liferay-ui:message key="group-statistics-add-counter-first-text" />
 	</span>
 
-	<aui:select inlineField="<%= true %>" label="" name='<%= portletResourceNamespace + "preferences--displayActivityCounterName" + index + "--" %>' title="display-activity-counter-name" useNamespace="<%= false %>">
+	<aui:select inlineField="<%= true %>" label="" name='<%= portletResourceNamespace + "displayActivityCounterName" + index %>' title="display-activity-counter-name" useNamespace="<%= false %>">
```
Comment (robertoDiaz) [inline modules/apps/collaboration/social/social-group-statistics-web/src/main/resources/META-INF/resources/add_activity_counter.jsp]: Hey @gregory-bretall 

I do not fully understand why do we need this change. 

What happens if the customer has stored a preference like "preferences--displayActivityCounterName-" and your changes are appliedv (I mean, deleting the "preferences--" part of the property) . Will the system retrieve the configured value or will return an empty/or default value??

**Comment by gregory-bretall** [comment]
Hi @robertoDiaz,
As far as my testing showed, the current situation is that for every single new auto field, a new separate preference row is added to the database. By removing the preferences--, what happens is that each preference is simply mapped to its correct key and should still be retrieved properly. The testing I did was with data from both before (with preferences--) and after (without preferences--) and it worked correctly. The change is mostly to clean up excess database rows although it isn't absolutely necessary to the fix working.

**Comment by robertoDiaz** [comment]
Ok, since it is not necessary, I'd prefer to avoid add noise to the pr and make it as small as possible.

So I'm going to resend it to you in order to rollback this change. Do you agree with it?

Thanks!!

**Comment by robertoDiaz** [comment]
Pull request submitted to https://github.com/gregory-bretall/liferay-portal/pull/4. [See changes here.](https://github.com/robertoDiaz/liferay-portal/compare/076dbe335427~1...076dbe335427) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by robertoDiaz** [comment]
Pull request submitted to https://github.com/adolfopa/liferay-portal/pull/668. [See changes here.](https://github.com/robertoDiaz/liferay-portal/compare/076dbe335427~1...076dbe335427) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**